Archipelago.MultiClient.Net/Colors/Color.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+
+namespace Archipelago.MultiClient.Net.Colors
+{
+    /// <summary>
+    /// Polyfill for System.Drawing.Color for games on runtimes which do not supply a `System.Drawing.dll`.
+    /// Not focused on memory or computation efficiency, we just need something that works reasonably well.
+    /// Transparency/alpha channel is not handled.
+    /// </summary>
+    public partial struct Color : IEquatable<Color>
+    {
+        public Color(byte r, byte g, byte b)
+        {
+            R = r;
+            G = g;
+            B = b;
+        }
+
+        public byte R { get; set; }
+
+        public byte G { get; set; }
+
+        public byte B { get; set; }
+
+        public override bool Equals(object obj)
```

Comment:
I dont think we need to implement the equals on struct, it should be implicit value equality

Reply:
Structs don't implicitly provide an implementation for `==` and `!=`. I thought this would be a good idea to include. Implementing `IEquatable<Color>` is just a bonus to ensure compatibility with other .NET APIs.

Reply:
Neat blog post I found on the topic: https://medium.com/@semuserable/c-journey-into-struct-equality-comparison-deep-dive-9693f74562f1